mirage/factories/milestone.js

```diff
@@ -12,6 +13,24 @@ export default Factory.extend({
   dcpMilestoneoutcome: null,
   milestoneLinks: [],
 
+  isCompleted: trait({
```

Comment:
All milestones that use the `isCompleted` trait would have the same `dcpActualstartdate`, and same for all milestones that use `isInProgress` or `isNotStarted`, which doesn't match real life? (a milestone's end date should be after the previous milestone's end date) Thinking that this might also affect/skew tests..

In the future maybe we should have fixtures for `before public review`, `in public review`  `in participant review` and `post review`  milestone lists. Each fixture would contain the entire list of milestones,  the milestone dates would be sequential, and we would assign `statuscode`s appropriate to the fixture. 

It'll be easier to explain in person